clients/rkecli/state.go

```diff
@@ -166,28 +165,27 @@ func NewStateFile(state *cluster.FullState, dirName string) (stateFilePath strin
 	return
 }
 
-// GetFullState is a function that gets RKE full state from "full-cluster-state" configmap.
+// GetFullState is a function that gets RKE full state from "full-cluster-state" secret.
 // And returns the cluster full state.
 func GetFullState(client *rancher.Client) (state *cluster.FullState, err error) {
-	namespacedConfigmapClient := client.Steve.SteveType(configmaps.ConfigMapSteveType).NamespacedSteveClient(cluster.SystemNamespace)
-	if err != nil {
-		return
-	}
+	namespacedSecretClient := client.Steve.SteveType("secret").NamespacedSteveClient(cluster.SystemNamespace)
```

Comment:
was configmap used for any version of 2.8 that we may need to not backport this to?

Reply:
This change was introduced in 2.8 versions of rancher and it needs to be back ported to release/v2.8.

Reply:
@markusewalker is this related to what we discussed offline back then?

Reply:
correct, replied offline.